massa-ledger/Cargo.toml

```diff
@@ -13,7 +13,7 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 thiserror = "1.0"
 tokio = { version = "1.11", features = ["full"] }
```

Comment:
Since here we depend on `tokio`, we need to propagate the `instrument` feature in this crate too!

Reply:
Done :)